frenchrubyshops.csv

```diff
@@ -50,5 +51,6 @@ Tymate,http://tymate.com/,agency,Lille
 Vodeclic,http://vodeclic.com/,product,Levallois
 Wayz-Up,https://www.wayz-up.com/,product,Paris
 Weroom,https://www.weroom.com/,product,Paris
+Wizville,https://wizville.fr/fr/,solution,Paris
```

Comment:
Not sure about the `solution` term. For consistency it would be better to have `product` as it's what Wizville is about (from what I've seen on your website).